models/User.js

```diff
@@ -1,5 +1,102 @@
 const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
+
+//Do we want start dates required?
+
+//Create education schema for use in ResumeSchema
+const EducationSchema = new Schema({
+  name:{
+    type: String,
+    required: true,
+  },
+  program:{
+    type: String,
+    required: true,
+  },
+  grade:{
+    type: Number,
+  },
+  start_date:{
+    type: Date,
+  },
+  end_date:{
+    type: Date,
+  }
+});
+
+//paid/volunteer experience schema for use in ResumeSchema
+const ExperienceSchema = new Schema({
+  title:{
+    type: String,
+    required: true,
+  },
+  company:{
+    type: String,
+    required: true,
+  },
+  start_date:{
+    type: Date,
+  },
+  end_date:{
+    type: Date,
+  },
+  description:{
+    type: String
+  }
+});
+
+//Project Schema for use in ResumeSchema
+const ProjectSchema = new Schema({
+  name:{
+    type: String,
+    required: true,
+  },
+  stack:{
+    type: String,
+  },
+  description:{
+    type: String,
+  },
+  links:[String]
+});
+
+const AwardSchema = new Schema({
+  name:{
+    type: String,
+    required: true,
+  },
+  description:{
+    type: String,
+    required: true,
+  },
+  date:{
+    type: Date,
+    required: true
+  }
+});
+
+//Create Resume Schema
+const ResumeSchema = new Schema({
+  name:{
+    type: String,
+    required: true,
+  },
+  email:{
+    type: String,
+    required: true,
+  },
+  phone_num:{
+    type: String,
+    required: true,
+  },
+  education:[EducationSchema],
+  skills:[String],
+  paid_experience:[ExperienceSchema],
+  volunteer_experience:[ExperienceSchema],
+  projects:[ProjectSchema],
+  awards:[AwardSchema]
```

Comment:
```suggestion
  awards:[AwardSchema],
  links:[{text: String, uri: String}]
```
It would be useful to have a links section here as well, that way we can put linked, github, personal websites, etc.

Reply:
having an array of links twice would be data redundancy, we should either do one way or the other.

Reply:
`ProjectSchema.links` will be for links related to specific projects. `ResumeSchema.links` will be personal links to put on the resume (like linkedin or your website). I don't see any way we could combine these collections and know which is which, so I don't think it is redundancy.

Reply:
oh ok, I didn't fully understand what you were getting at thanks for clarifying that's my bad.